app/domain/trips/stats/weekly_stats.rb

```diff
@@ -0,0 +1,19 @@
+module Trips
+  module Stats
+    class WeeklyStats < ActiveModelSerializers::Model
+      def total_distance
+        trips.sum(&:distance)
+      end
+
+      def total_price
+        trips.sum(&:price)
+      end
+
+      private
+
+      def trips
+        @_trips ||= Trip.where(date: Date.current.all_week)
+      end
```

Comment:
Here i decided to do single query and calculate in memory. If there was a lot of data we could use `select(:distance, :price)` on trips to not load them fully, or simply let db calculate it like below
```suggestion
      def total_distance
        trips.sum(:distance)
      end

      def total_price
        trips.sum(:price)
      end

      private

      def trips
        Trip.where(date: Date.current.all_week)
      end
```